apps/codeforafrica/src/components/NavBar/DesktopNavigation/index.js

```diff
@@ -0,0 +1,50 @@
+import { Grid } from "@mui/material";
+import PropTypes from "prop-types";
+import React from "react";
+
+/* eslint-disable import/no-unresolved */
+import cfaLogo from "@/codeforafrica/assets/CfA logo.svg?url";
+import ImageIcon from "@/codeforafrica/components/ImageIcon";
+import NavBarNavList from "@/codeforafrica/components/NavBarNavList";
+
+const DesktopNavigation = React.forwardRef(function DesktopNavigation(
+  props,
+  ref
+) {
+  const { menu } = props;
+
+  return (
+    <Grid
+      container
+      justifyContent="space-between"
+      alignItems="center"
+      direction="row"
+      ref={ref}
+      sx={{ display: { xs: "none", md: "flex" } }}
```

Comment:
```suggestion
```
This should be set in the main `NavBar` i.e. show `MobileNavBar` in xs/sm and show `DesktopNavBar` if `md` and above